savedefconfig_to_baseconfig.sh

```diff
@@ -0,0 +1,21 @@
+if [ -z "$1" ] || [ -z "$2" ]
+    then
+        echo "Usage: ./savedefconfig_to_baseconfig.sh defconfig output_name"
+	exit 1
+fi
+
+echo "Processing file: $1"
+echo "Output file: $2"
+
+if [ -f "$1" ]; then
+    cat android-base.config \
+        android-recommended.config \
+        android-recommended-arm64.config > \
+        /tmp/android_recommended && \
+    sort -o /tmp/android_recommended /tmp/android_recommended && \
```

Comment:
You know that `<(some command)` also works here for `<(cat android-{base,recommended,recommended-arm64}.config)`?